presto-docs/src/main/sphinx/release/release-0.251.rst

```diff
@@ -0,0 +1,34 @@
+=============
+Release 0.251
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve dictionary writer performance.
```

Comment:
```suggestion
* Improve ORC dictionary writer performance.
```

Reply:
In my opinion - if we cannot clearly point out the details of the improvement - how much improvement or what kind of queries it might help - we should not put this point in the release notes

Reply:
My guess is any table that has varchar/string contents would be improved.  How about

`Improve ORC writer performance for string columns`